xbmc/video/VideoThumbLoader.cpp

```diff
@@ -86,6 +86,9 @@ bool CThumbExtractor::operator==(const CJob* job) const
 bool CThumbExtractor::DoWork()
 {
   if (m_item.IsLiveTV()
```

Comment:
> Doesn't this include recordings?

No. Definitely not. I checked the implementation of IsLiveTV() and ofc did runtime tests.